source/ca.mover.tuning/usr/local/emhttp/plugins/ca.mover.tuning/updateCron.php

```
@@ -20,9 +23,26 @@ function logger($string)
 	}
 }
 
+function make_tune_cron()
+{
+	global $vars;
+	if (!empty($var['shareMoverSchedule'])) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$var'. (undefined)

(UnusedLocalVariable)